subscribers:
  command_base_path:
    topic: /mpc_trajectory
    queue_size: 1

publishers:
  current_pose:
    topic: /smb_path_following_controller/current_pose
  optimal_pose:
    topic: /smb_path_following_controller/optimal_pose
  optimal_path:
    topic: /smb_path_following_controller/optimal_path

ros_publishing_rate: 10 # [Hz]